### PR TITLE
Updates in installation instructions in the V1 documentation

### PR DESCRIPTION
**Pull request by kkovlakas**
- [x] Adding introduction in the installation instructions, explaining the two ways to install/download data, and why we care.
- [x] Restructuring the sections
- [x] Repeating the download-script instruction in the conda installation (and the `bashrc` trick)
- [x] Adding the `git checkout main` instruction before pip installing from github (V1)
- [x] Language check
- [x] Final move to the website (using the `github.io`

### CODE REVIEW

File: docs/install/install.rst [hunk 1]
```diff
@@ -4,22 +4,36 @@
...
-conda environment can be created (we have named our environment posydon-example, 
-but you can choose any name), the conda-forge channel added, and the required 
+The easiest way to start with POSYDON is to install it from Anaconda
+(Automatic Installation), and call an executable script that downloads the grid
+data automatically (Automatic Data Download).
+
+Alternatively, if you are interested in modifying the code, you can install
+POSYDON manually through `github` (Manual Installation). You can still use the
+automatic script to download the data, however, if you are about to modify the
+grid data, you can get them using the `git-lfs` module (Manual Data Download).
```
Comment (mkruckow) [inline docs/install/install.rst]: Is the git-lfs still in place? I remember Simone told me once, this isn't available anymore. On the other hand, do we need this for v1? Shouldn't we there always use the data on Zenodo?

**Comment by mkruckow** [comment]
I think, in the automatic installation now there is the part about exporting the global path missing.
`export PATH_TO_POSYDON=/home/POSYDON/`
or is this automatically done?
If it is done automatically, is it done only for the session or added to the `.bash_rc` or `.bash_profile`? I haven't seen anything in the repo doing this.
Finally, I'm wondering, whether we should have a default for `export PATH_TO_POSYDON_DATA=/home/` or `export PATH_TO_POSYDON_DATA=$PATH_TO_POSYDON/data/`, in the case of automatically doing one export to do all.

**Comment by astroJeff** [comment]
I just reviewed the language. I made some changes (not just to your text, but what was there before. Please read through to make sure you agree with my updates.

**Review by tassos25 — APPROVED** [review]
Looks good

**Comment by maxbriel** [comment]
These changes look good. I would recommend adding the following information to the Mac section, because many packages for python3.7 are not available on osx-arm

After creating an environment the user has to change the architecture channels. This will automatically enable Rosetta to translate the osx64 architecture into osx-arm. Although many things should just work, I expect it to reduced performance and I have not tested this extensively to see if anything breaks. 

```
conda create --name posydon_conda

conda activate posydon_conda

 ## use x86_64 architecture channel(s)
conda config --env --set subdir osx-64

conda install python=3.7

conda install posydon

```

**Comment by kkovlakas** [comment]
> I think, in the automatic installation now there is the part about exporting the global path missing. `export PATH_TO_POSYDON=/home/POSYDON/` or is this automatically done? If it is done automatically, is it done only for the session or added to the `.bash_rc` or `.bash_profile`? I haven't seen anything in the repo doing this. Finally, I'm wondering, whether we should have a default for `export PATH_TO_POSYDON_DATA=/home/` or `export PATH_TO_POSYDON_DATA=$PATH_TO_POSYDON/data/`, in the case of automatically doing one export to do all.

Thank you for considering this @mkruckow . Actually, `PATH_TO_POSYDON` is only used in unit-testing and running grids, so we define it only as developers. Users installing via `conda` or `pip install` won't need it. Although there is a variable defined in `common_function.py` which read the environment variable, it's empty if undefined, and POSYDON works normally.

**Comment by kkovlakas** [comment]
> These changes look good. I would recommend adding the following information to the Mac section, because many packages for python3.7 are not available on osx-arm
> 
> After creating an environment the user has to change the architecture channels. This will automatically enable Rosetta to translate the osx64 architecture into osx-arm. Although many things should just work, I expect it to reduced performance and I have not tested this extensively to see if anything breaks.
> 
> ```
> conda create --name posydon_conda
> 
> conda activate posydon_conda
> 
>  ## use x86_64 architecture channel(s)
> conda config --env --set subdir osx-64
> 
> conda install python=3.7
> 
> conda install posydon
> ```

Thank you @maxbriel , I add this extra set of instruction -however, I am not sure about conda channels, can I include them?

**Comment by maxbriel** [comment]
I've added the additional conda channels to the instructions, although I don't think they're explicitly necessary to define.

**Review by maxbriel — APPROVED** [review]

**Pull request merged**